[2017-11-12 11:11:52] DEBUG [mediconnect.index:13] Entering index method
[2017-11-12 20:45:43] INFO [django.server.log_message:124] "GET / HTTP/1.1" 200 1971
[2017-11-12 20:45:44] INFO [django.server.log_message:124] "GET /static/js/jquery.twbsPagination.js HTTP/1.1" 200 11819
[2017-11-12 20:45:44] INFO [django.server.log_message:124] "GET /static/js/bootstrap.min.js HTTP/1.1" 200 37051
[2017-11-12 20:45:44] INFO [django.server.log_message:124] "GET /static/js/jquery.js HTTP/1.1" 200 86711
[2017-11-12 20:45:44] INFO [django.server.log_message:124] "GET /static/css/bootstrap.min.css HTTP/1.1" 200 121205
[2017-11-12 20:45:44] INFO [django.server.log_message:124] "GET /static/js/vue.js HTTP/1.1" 200 279117
[2017-11-12 20:45:44] WARNING [django.server.log_message:124] "GET /static/js/table2CSV.js HTTP/1.1" 404 1772
[2017-11-12 20:45:44] INFO [django.server.log_message:124] "GET /static/js/jquery-ui.js HTTP/1.1" 200 539419
[2017-11-12 20:45:44] INFO [django.server.log_message:124] "GET /jsreverse/ HTTP/1.1" 200 7485
[2017-11-12 20:45:45] WARNING [django.server.log_message:124] "GET /static/js/table2CSV.js HTTP/1.1" 404 1772
[2017-11-12 20:45:45] WARNING [django.request.get_response:152] Not Found: /favicon.ico
[2017-11-12 20:45:45] WARNING [django.server.log_message:124] "GET /favicon.ico HTTP/1.1" 404 4140
[2017-11-12 20:45:46] INFO [django.server.log_message:124] "GET /supervisor/translator_list/13/ HTTP/1.1" 200 3502
[2017-11-12 20:45:46] WARNING [django.server.log_message:124] "GET /static/js/table2CSV.js HTTP/1.1" 404 1772
[2017-11-12 20:45:46] INFO [django.server.log_message:124] "GET /jsreverse/ HTTP/1.1" 200 7485
[2017-11-12 20:45:46] WARNING [django.server.log_message:124] "GET /static/js/table2CSV.js HTTP/1.1" 404 1772
[2017-11-12 20:46:14] INFO [django.server.log_message:124] "GET /supervisor/translator_list/13/ HTTP/1.1" 200 3502
[2017-11-12 20:46:14] WARNING [django.server.log_message:124] "GET /static/js/table2CSV.js HTTP/1.1" 404 1772
[2017-11-12 20:46:14] INFO [django.server.log_message:124] "GET /jsreverse/ HTTP/1.1" 200 7485
[2017-11-12 20:46:14] WARNING [django.server.log_message:124] "GET /static/js/table2CSV.js HTTP/1.1" 404 1772
[2017-11-12 20:46:16] INFO [django.server.log_message:124] "GET /supervisor/orders/13/All/ HTTP/1.1" 200 9603
[2017-11-12 20:46:16] WARNING [django.server.log_message:124] "GET /static/js/table2CSV.js HTTP/1.1" 404 1772
[2017-11-12 20:46:16] INFO [django.server.log_message:124] "GET /jsreverse/ HTTP/1.1" 200 7485
[2017-11-12 20:46:16] WARNING [django.server.log_message:124] "GET /static/js/table2CSV.js HTTP/1.1" 404 1772
[2017-11-12 20:46:17] INFO [django.server.log_message:124] "GET /supervisor/ajax/update_result/?status=All&query=%7B%7D HTTP/1.1" 200 1672
[2017-11-12 20:46:17] INFO [django.server.log_message:124] "GET /supervisor/ajax/update_result/?status=All&query=%7B%7D&page=1 HTTP/1.1" 200 1672
[2017-11-13 09:01:37] INFO [django.server.log_message:124] "GET /supervisor/orders/13/All/ HTTP/1.1" 200 9603
[2017-11-13 09:01:37] WARNING [django.server.log_message:124] "GET /static/js/table2CSV.js HTTP/1.1" 404 1772
[2017-11-13 09:01:37] INFO [django.server.log_message:124] "GET /jsreverse/ HTTP/1.1" 200 7485
[2017-11-13 09:01:38] WARNING [django.server.log_message:124] "GET /static/js/table2CSV.js HTTP/1.1" 404 1772
[2017-11-13 09:01:38] INFO [django.server.log_message:124] "GET /supervisor/ajax/update_result/?status=All&query=%7B%7D HTTP/1.1" 200 1672
[2017-11-13 09:01:38] INFO [django.server.log_message:124] "GET /supervisor/ajax/update_result/?status=All&query=%7B%7D&page=1 HTTP/1.1" 200 1672
[2017-11-13 09:01:39] INFO [django.server.log_message:124] "GET /supervisor/translator_list/13/ HTTP/1.1" 200 3503
[2017-11-13 09:01:39] WARNING [django.server.log_message:124] "GET /static/js/table2CSV.js HTTP/1.1" 404 1772
[2017-11-13 09:01:39] INFO [django.server.log_message:124] "GET /jsreverse/ HTTP/1.1" 200 7485
[2017-11-13 09:01:39] WARNING [django.server.log_message:124] "GET /static/js/table2CSV.js HTTP/1.1" 404 1772
[2017-11-13 09:01:43] ERROR [django.request.handle_uncaught_exception:135] Internal Server Error: /supervisor/ajax/send_reset_link/
Traceback (most recent call last):
  File "C:\Python27\lib\site-packages\django\core\handlers\exception.py", line 41, in inner
    response = get_response(request)
  File "C:\Python27\lib\site-packages\django\core\handlers\base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\Python27\lib\site-packages\django\core\handlers\base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Python27\lib\site-packages\django\contrib\auth\decorators.py", line 23, in _wrapped_view
    return view_func(request, *args, **kwargs)
  File "C:\Users\gabri\Desktop\Project\website\medi_connect_site\supervisor\views.py", line 423, in send_reset_link
    user = User.objects.get(id = trans_id)
  File "C:\Python27\lib\site-packages\django\db\models\manager.py", line 85, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "C:\Python27\lib\site-packages\django\db\models\query.py", line 380, in get
    self.model._meta.object_name
DoesNotExist: User matching query does not exist.
[2017-11-13 09:01:43] ERROR [django.server.log_message:124] "GET /supervisor/ajax/send_reset_link/?trans_id=42&password=1234567 HTTP/1.1" 500 17933
[2017-11-13 09:04:07] INFO [django.server.log_message:124] "GET /supervisor/translator_list/13/ HTTP/1.1" 200 3503
[2017-11-13 09:04:07] WARNING [django.server.log_message:124] "GET /static/js/table2CSV.js HTTP/1.1" 404 1772
[2017-11-13 09:04:07] INFO [django.server.log_message:124] "GET /jsreverse/ HTTP/1.1" 200 7485
[2017-11-13 09:04:07] WARNING [django.server.log_message:124] "GET /static/js/table2CSV.js HTTP/1.1" 404 1772
[2017-11-13 09:04:16] ERROR [django.request.handle_uncaught_exception:135] Internal Server Error: /supervisor/ajax/send_reset_link/
Traceback (most recent call last):
  File "C:\Python27\lib\site-packages\django\core\handlers\exception.py", line 41, in inner
    response = get_response(request)
  File "C:\Python27\lib\site-packages\django\core\handlers\base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\Python27\lib\site-packages\django\core\handlers\base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Python27\lib\site-packages\django\contrib\auth\decorators.py", line 23, in _wrapped_view
    return view_func(request, *args, **kwargs)
  File "C:\Users\gabri\Desktop\Project\website\medi_connect_site\supervisor\views.py", line 468, in send_reset_link
    email_message.send()
  File "C:\Python27\lib\site-packages\django\core\mail\message.py", line 348, in send
    return self.get_connection(fail_silently).send_messages([self])
  File "C:\Python27\lib\site-packages\django\core\mail\backends\smtp.py", line 104, in send_messages
    new_conn_created = self.open()
  File "C:\Python27\lib\site-packages\django\core\mail\backends\smtp.py", line 71, in open
    self.connection.login(force_str(self.username), force_str(self.password))
  File "C:\Python27\lib\smtplib.py", line 622, in login
    raise SMTPAuthenticationError(code, resp)
SMTPAuthenticationError: (451, '4.7.0 Temporary server error. Please try again later. PRX4  [MWHPR22CA0071.namprd22.prod.outlook.com]')
[2017-11-13 09:04:16] ERROR [django.server.log_message:124] "GET /supervisor/ajax/send_reset_link/?trans_id=42&password=1234567 HTTP/1.1" 500 18372
[2017-11-13 09:06:42] INFO [django.server.log_message:124] "GET /supervisor/customer_list/13/ HTTP/1.1" 200 4460
[2017-11-13 09:06:42] WARNING [django.server.log_message:124] "GET /static/js/table2CSV.js HTTP/1.1" 404 1772
[2017-11-13 09:06:42] INFO [django.server.log_message:124] "GET /jsreverse/ HTTP/1.1" 200 7485
[2017-11-13 09:06:42] WARNING [django.server.log_message:124] "GET /static/js/table2CSV.js HTTP/1.1" 404 1772
[2017-11-13 09:06:43] INFO [django.server.log_message:124] "GET /supervisor/translator_list/13/ HTTP/1.1" 200 3503
[2017-11-13 09:06:44] WARNING [django.server.log_message:124] "GET /static/js/table2CSV.js HTTP/1.1" 404 1772
[2017-11-13 09:06:44] INFO [django.server.log_message:124] "GET /jsreverse/ HTTP/1.1" 200 7485
[2017-11-13 09:06:44] WARNING [django.server.log_message:124] "GET /static/js/table2CSV.js HTTP/1.1" 404 1772
[2017-11-13 09:06:46] INFO [django.server.log_message:124] "GET /supervisor/orders/13/All/ HTTP/1.1" 200 9603
[2017-11-13 09:06:46] WARNING [django.server.log_message:124] "GET /static/js/table2CSV.js HTTP/1.1" 404 1772
[2017-11-13 09:06:46] INFO [django.server.log_message:124] "GET /jsreverse/ HTTP/1.1" 200 7485
[2017-11-13 09:06:46] WARNING [django.server.log_message:124] "GET /static/js/table2CSV.js HTTP/1.1" 404 1772
[2017-11-13 09:06:46] INFO [django.server.log_message:124] "GET /supervisor/ajax/update_result/?status=All&query=%7B%7D HTTP/1.1" 200 1672
[2017-11-13 09:06:46] INFO [django.server.log_message:124] "GET /supervisor/ajax/update_result/?status=All&query=%7B%7D&page=1 HTTP/1.1" 200 1672
[2017-11-13 09:49:44] INFO [django.server.log_message:124] "GET / HTTP/1.1" 200 1971
[2017-11-13 09:49:44] WARNING [django.server.log_message:124] "GET /static/js/table2CSV.js HTTP/1.1" 404 1772
[2017-11-13 09:49:44] INFO [django.server.log_message:124] "GET /jsreverse/ HTTP/1.1" 200 7485
[2017-11-13 09:49:45] WARNING [django.server.log_message:124] "GET /static/js/table2CSV.js HTTP/1.1" 404 1772
[2017-11-13 09:49:48] INFO [django.server.log_message:124] "GET /supervisor/orders/13/All/ HTTP/1.1" 200 9603
[2017-11-13 09:49:48] WARNING [django.server.log_message:124] "GET /static/js/table2CSV.js HTTP/1.1" 404 1772
[2017-11-13 09:49:48] INFO [django.server.log_message:124] "GET /jsreverse/ HTTP/1.1" 200 7485
[2017-11-13 09:49:48] WARNING [django.server.log_message:124] "GET /static/js/table2CSV.js HTTP/1.1" 404 1772
[2017-11-13 09:49:48] INFO [django.server.log_message:124] "GET /supervisor/ajax/update_result/?status=All&query=%7B%7D HTTP/1.1" 200 1638
[2017-11-13 09:49:48] INFO [django.server.log_message:124] "GET /supervisor/ajax/update_result/?status=All&query=%7B%7D&page=1 HTTP/1.1" 200 1638
[2017-11-13 09:52:08] INFO [django.server.log_message:124] "GET /supervisor/export_csv/ HTTP/1.1" 200 37
[2017-11-13 20:18:02] INFO [django.server.log_message:124] "GET / HTTP/1.1" 200 1971
[2017-11-13 20:18:03] WARNING [django.server.log_message:124] "GET /static/js/table2CSV.js HTTP/1.1" 404 1772
[2017-11-13 20:18:03] INFO [django.server.log_message:124] "GET /jsreverse/ HTTP/1.1" 200 7485
[2017-11-13 20:18:03] WARNING [django.server.log_message:124] "GET /static/js/table2CSV.js HTTP/1.1" 404 1772
[2017-11-13 20:18:04] WARNING [django.request.get_response:152] Not Found: /favicon.ico
[2017-11-13 20:18:04] WARNING [django.server.log_message:124] "GET /favicon.ico HTTP/1.1" 404 4140
[2017-11-13 20:18:04] INFO [django.server.log_message:124] "GET /supervisor/translator_list/13/ HTTP/1.1" 200 4105
[2017-11-13 20:18:05] WARNING [django.server.log_message:124] "GET /static/js/table2CSV.js HTTP/1.1" 404 1772
[2017-11-13 20:18:05] INFO [django.server.log_message:124] "GET /jsreverse/ HTTP/1.1" 200 7485
[2017-11-13 20:18:05] WARNING [django.server.log_message:124] "GET /static/js/table2CSV.js HTTP/1.1" 404 1772
[2017-11-13 20:18:06] INFO [django.server.log_message:124] "GET /supervisor/ajax/send_reset_link/?trans_id=42&password= HTTP/1.1" 200 66
